packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -145,11 +145,11 @@ export class EpochTracker {
     }
 
     private checkForEpochError(error) {
-        assert(!!this._hashedDocumentId, "DocId should be set to clear the cached entries!!");
         if (error.errorType === OdspErrorType.epochVersionMismatch) {
             if (this.logger !== undefined) {
                 this.logger.sendErrorEvent({ eventName: "EpochVersionMismatch" }, error);
             }
+            assert(!!this._hashedDocumentId, "DocId should be set to clear the cached entries!!");
```

Comment:
I assume this is done due to create new path, right?

Reply:
yes